src/server/game/Entities/Item/ItemTemplate.h

```diff
@@ -415,6 +415,20 @@ enum InventoryType : uint8
 
 #define MAX_INVTYPE                               35
 
+constexpr std::array<InventoryType, 10> InventoryTypesEquipable =
```

Comment:
Eh, making it local in ObjectMgr::LoadEquipmentTemplates seems fine (I don't see another use for this)

Reply:
True, I don't know why I was thinking about validations for SAI. Fixed

Reply:
Maybe because SAI can set random items (not validated) if action_param1 = 0

Reply:
Sooo... should I move it back to where I put it? I don't like the idea of being able to try to equip any item in that SAI action